### ISSUE TITLE
Windows builds are missing the Universal CRT ApiSet Stub DLLs (api-ms-win-*.dll)

### ISSUE THREAD

**Issue by bitraid** [qutebrowser/qutebrowser#3866]
These missing files will cause qutebrowser to fail to start on Windows versions older than Windows 10, that don't have [KB2999226](https://support.microsoft.com/en-us/help/2999226/update-for-universal-c-runtime-in-windows) installed. 
Pyinstaller warns about the missing libraries:

```WARNING: lib not found: api-ms-win-crt-heap-l1-1-0.dll dependency of z:\prj\qutebrowser\.tox\pyinstaller\scripts\python.exe
WARNING: lib not found: api-ms-win-crt-stdio-l1-1-0.dll dependency of z:\prj\qutebrowser\.tox\pyinstaller\scripts\python.exe
WARNING: lib not found: api-ms-win-crt-runtime-l1-1-0.dll dependency of z:\prj\qutebrowser\.tox\pyinstaller\scripts\python.exe
WARNING: lib not found: api-ms-win-crt-math-l1-1-0.dll dependency of z:\prj\qutebrowser\.tox\pyinstaller\scripts\python.exe
WARNING: lib not found: api-ms-win-crt-locale-l1-1-0.dll dependency of z:\prj\qutebrowser\.tox\pyinstaller\scripts\python.exe
WARNING: lib not found: api-ms-win-crt-string-l1-1-0.dll dependency of C:\Windows\system32\vcruntime140.dll
WARNING: lib not found: api-ms-win-crt-heap-l1-1-0.dll dependency of C:\Windows\system32\vcruntime140.dll
...
```
There are 39 files needed, but these files are small - less than 1mb in total size.
I think that these files should be included, so users can run qutebrowser on fresh installs of previous Windows versions (or on systems missing the update) without problems. Other programs (firefox) seem to include these files, too.

I would suggest the build script to add `%ProgramFiles(x86)%\Windows Kits\10\Redist\ucrt\DLLs\x86` and `%ProgramFiles(x86)%\Windows Kits\10\Redist\ucrt\DLLs\x64` (The files are part of the [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk)) to path, and maybe throw a warning if not found.

**Comment by The-Compiler** [comment]
Duplicate of #2788?

**Comment by bitraid** [comment]
Yes, that seems like a missing/corrupt ucrt problem. Microsoft refactored their c runtimes in 2015 calling it [Universal CRT](https://blogs.msdn.microsoft.com/vcblog/2015/03/03/introducing-the-universal-crt/) and made it part of the OS, in hopes to address the DLL hell. But that gave rise to `The program can’t start because api-ms-win-crt-runtime-l1-1-0.dll is missing` problem, thats why i'd recommend to include the DLLs.

**Comment by joshodell** [comment]
Did including these DLLs fix the issue?

**Comment by The-Compiler** [comment]
Installing the Windows SDK right now, let's see whether that causes PyInstaller to include those files automatically.

**Comment by bitraid** [comment]
The `PATH` must be manualy updated (better be the first in list).

**Comment by The-Compiler** [comment]
Yup, seems to work, so those will be included in the v1.6.0 release!
